drivers/wireless/ieee80211/bcm43xxx/bcmf_bdc.c

```diff
@@ -48,30 +48,30 @@
  * Private Types
  ****************************************************************************/
 
-struct __attribute__((packed)) bcmf_bdc_header
+begin_packed_struct struct bcmf_bdc_header
```

Comment:
let's #4238 fix this?

Reply:
No problem, let's do it in #4238. I've already replaced __ attribute __ occurrences here in #4236 PR (in my branch) for bcm43xxx.
As I understand, it should be normally merged anyway. If I should remove the changes in my branch, please let me know.